modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -478,3 +426,75 @@ class Configuration(object):
             url = url.replace("{" + variable_name + "}", used_value)
 
         return url
+
+class HttpSigningConfiguration(object):
+    """NOTE: This class is auto generated by OpenAPI Generator
+
+    Ref: https://openapi-generator.tech
+    Do not edit the class manually.
+
+    :param key_id: The identifier of the cryptographic key, when signing HTTP requests.
```

Comment:
Can you make sure that the signing_info dict is described here and that all of its subkeys and values are also describe here?